basiclti/basiclti-blis/src/java/org/sakaiproject/lti13/LTI13Servlet.java

```diff
@@ -1530,4 +1536,45 @@ private void handleLineItemsDelete(String signed_placement, String lineItem, Htt
 		LTI13Util.return400(response, "Could not delete assignment "+assignment_id);
 		log.error("Could delete assignment={}", assignment_id);
 	}
+
+	/**
+	 * Process the returned OIDC Authorization request
+	 * @param signed_placement
+	 * @param lineItem - Can be null
+	 * @param results
+	 * @param request
+	 * @param response
+	 */
+	private void handleOIDCAuthorization(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+		String state = (String) request.getParameter("state");
+		if ( state != null && state.trim().length() < 1 ) state = null;
```

Comment:
StringUtils.trimToNull does all of this hard work for you!

Reply:
I guess.  At times, I tire of using things that are not in Java normally.  Java should fix themselves. :)